src/main/java/dan200/computercraft/shared/peripheral/speaker/SoundTask.java

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
I've never found a need to put an anonymous class on the constructor. Just "construct" things in the enclosing scope.